RecDP/pyrecdp/primitives/operations/text_pii_remove.py

```diff
@@ -30,8 +45,10 @@ def process(sample):
 
 
 class PIIRemoval(BaseLLMOperation):
-    def __init__(self, text_key = 'text', inplace = True, model_root_path = "", debug_mode = False):
-        settings = {'text_key': text_key, 'inplace': inplace, 'model_root_path': model_root_path, 'debug_mode': debug_mode}
+    def __init__(self, text_key='text', inplace=True, model_root_path="", debug_mode=False,
+                 entity_types: List[PIIEntityType] = None):
+        settings = {'text_key': text_key, 'inplace': inplace, 'model_root_path': model_root_path,
+                    'debug_mode': debug_mode}
```

Comment:
@chaojun-zhang , entity_types need to put inside settings. settings will be used to init super() class